### PR TITLE
Feature/finish mvp

### PR DESCRIPTION
**Pull request by mattmcvey**
## Is this a fix or a feature?
Feature
## What is the change?
Added ability for random meals to display and cookpot to disappear.
## What does it fix?
No fix a feature.
## Where should the reviewer start?
The main.js file has all of the new javascript for functionality.
## How should this be tested?
Testing functionality of the site.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  event.preventDefault();
+  var checkedDish;
+  var foods = document.querySelectorAll("input[name='food']");
+  for(var i = 0; i < foods.length; i++){
+    if(foods[i].checked){
+      checkedDish = foods[i].value;
+      break;
+    };
+  };
+  if(checkedDish === "side"){
```
Comment (kevinhartmann23) [inline main.js:20]: consider splitting this function up? ex: checkRadioButtons() / displayText() 
Usual naming conventions on functions are named after what it is the function is executing....

File: main.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+var cookpot = document.querySelector(".cookpot");
+var rightBox = document.querySelector(".right-box");
+
+
+letsCookButton.addEventListener('click', checkingRadioButtons)
+
+function checkingRadioButtons() {
+  event.preventDefault();
+  var checkedDish;
+  var foods = document.querySelectorAll("input[name='food']");
```
Comment (kevinhartmann23) [inline main.js:13]: Overall, it looks good. I would not have thought to do it this way, but nice work! That's the beauty of coding.
Potential Refactor - 
- think about what a <input> radio tag is, it has checked value that is true/false
- they have values, id's, names, etc.... could using querySelectors on this buttons provide more information you need to shorten your code, and potentially allow you to remove lines 12-18?

**Review by kevinhartmann23 — COMMENTED** [review]
Looks good Matt, if you need or would like to discuss any comments feel free to reach out!

File: main.js (hunk 2 above)
Comment (kevinhartmann23) [reply main.js:13]: Traditionally the name assigned to an input is for accessibility (ie. screen readers), and would want seperates names dedicated to them. 

name="sides"
name="thing"
etc.

File: main.js (hunk 1 above)
Comment (mattmcvey) [reply main.js:20]: Thanks Kevin. This is exactly what I have been trying to figure out how to do is split into 2 functions. Haven't quite gotten there yet...

**Pull request merged**